### PR TITLE
Update release related information

### PR DESCRIPTION
**Pull request by atalman**
Update release related information. Features became more complex. Number of commits per releases have increased a lot.
We had in average:
2.5k commits for releases 1.1.0-1.7.0, 
3-3.5k commits for releases  1.8.0-1.12.0
4.5k-5k commits for releases  1.13.0, 2.0.0

Hence current target is 3 releases a year

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,14 +434,14 @@ Three-pointers to get you started:
 
 ## Releases and Contributing
 
-PyTorch has a 90-day release cycle (major releases). Please let us know if you encounter a bug by [filing an issue](https://github.com/pytorch/pytorch/issues).
+PyTorch has 3 releases a year (major releases). Please let us know if you encounter a bug by [filing an issue](https://github.com/pytorch/pytorch/issues).
```
Comment (svekars) [inline README.md:437]: typically
```suggestion
Typically, PyTorch has three major releases a year. Please let us know if you encounter a bug by [filing an issue](https://github.com/pytorch/pytorch/issues).
```

File: README.md [hunk 2]
```diff
@@ -434,14 +434,14 @@ Three-pointers to get you started:
...
 
 We appreciate all contributions. If you are planning to contribute back bug-fixes, please do so without any further discussion.
 
 If you plan to contribute new features, utility functions, or extensions to the core, please first open an issue and discuss the feature with us.
 Sending a PR without discussion might end up resulting in a rejected PR because we might be taking the core in a different direction than you might be aware of.
 
-To learn more about making a contribution to Pytorch, please see our [Contribution page](CONTRIBUTING.md).
+To learn more about making a contribution to Pytorch, please see our [Contribution page](CONTRIBUTING.md). For release related information consult [Release page](RELEASE.md).
```
Comment (svekars) [inline README.md:444]:
```suggestion
To learn more about making a contribution to Pytorch, please see our [Contribution page](CONTRIBUTING.md). For more information about PyTorch releases, see [Release page](RELEASE.md).
```

**Review by svekars — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (malfet) [inline README.md:437]: Would be good to provide a justification, why frequency was reduce from 4 times a year, to 3

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**